lib/src/services/databases/calendar/implementation.dart

```diff
@@ -68,6 +69,9 @@ class LocalCalendar implements CalendarInterface {
 		.getAll(Idb.scheduleStoreName);
 
 	@override
+	Future<Map<String, String>> getDefaultSchedules() => Idb.instance
+			.get("schedules","Monday") as Future<Map<String, String>>;
```

Comment:
```suggestion
			.get("schedules","Monday");
```

1. No `as`, see above
2. Why Monday?